### ISSUE TITLE
CAN'T RUN ON WINDOWS LOCAL MACHINE

### ISSUE THREAD

**Issue by aaron2181** [JeffreyCA/spleeter-web#5]
Hi, I have followed all the procedures in Readme.md for running the app both with docker and without docker. Everything seems fine, no errors but http://0.0.0.0:8000 in my browser gives "This site can’t be reached".

Also, can it be deployed on VPS? How?

Thanks

**Comment by JeffreyCA** [comment]
Can you try http://localhost:8000 or http://127.0.0.1:8000? Do you get the issue?

**Comment by JeffreyCA** [comment]
To deploy on VPS I recommend using Docker. Just make sure to install Docker and Docker Compose on the VPS and then follow the Deploying instructions in README.

You may have to tweak [settings_docker.py](https://github.com/JeffreyCA/spleeter-web/blob/master/django_react/settings_docker.py) to change to your preferred storage backend. Right now I can only confirm it works with Azure storage, I'll have to test it works with others.

**Comment by aaron2181** [comment]
> Can you try http://localhost:8000 or http://127.0.0.1:8000? Do you get the same issue?

Yes, I have tried them but getting the same issue. I am on windows 10.

**Comment by JeffreyCA** [comment]
Can you confirm you have the latest changes and you ran this Docker command?
`docker-compose -f docker-compose.yml -f docker-compose.build.yml -f docker-compose.dev.yml up --build`.

Can you also share the logs after:
```
Creating spleeter-web_frontend_1 ... done
Creating spleeter-web_redis_1    ... done
Creating spleeter-web_db_1       ... done
Creating spleeter-web_huey_1     ... done
Creating spleeter-web_api_1      ... done
Creating spleeter-web_nginx_1    ... done
```

**Comment by aaron2181** [comment]
![error](https://user-images.githubusercontent.com/47740236/84569180-07639600-ad74-11ea-8dbc-8b215283ff8f.PNG)


what I get by running docker

D:\54vibesApps\splitvibes>docker-compose -f docker-compose.yml -f docker-compose.build.yml -f docker-compose.dev.yml up --build
Creating network "splitvibes_default" with the default driver
Building frontend
Step 1/11 : FROM node:12-buster
 ---> 1f26404403b8
Step 2/11 : RUN mkdir -p /webapp/frontend/assets
 ---> Using cache
 ---> dcd2c92bba4e
Step 3/11 : WORKDIR /webapp/frontend
 ---> Using cache
 ---> c770fde1c276
Step 4/11 : COPY package.json /webapp/frontend/
 ---> Using cache
 ---> 1e21a7699939
Step 5/11 : COPY package-lock.json /webapp/frontend/
 ---> Using cache
 ---> 2a6c9f73402b
Step 6/11 : RUN npm install
 ---> Running in 4bf7f12b9729
npm WARN spleeter-web-frontend@1.0.0 No description
npm WARN spleeter-web-frontend@1.0.0 No repository field.

npm ERR! network timeout at: https://registry.npmjs.org/@babel/helper-plugin-utils/-/helper-plugin-utils-7.8.3.tgz

**Comment by JeffreyCA** [comment]
Looks like a network issue. I suggest checking your firewall configuration and Docker network settings.

**Comment by aaron2181** [comment]
I have a disabled firewall but still, the same issue both docker and without docker.

**Comment by JeffreyCA** [comment]
So you're seeing the error when running `npm install` without Docker? Can you access the file at https://registry.npmjs.org/@babel/helper-plugin-utils/-/helper-plugin-utils-7.8.3.tgz in  your browser?

**Comment by aaron2181** [comment]
I don't see the error when running npm install without Docker. 
Yes, I can access the file at https://registry.npmjs.org/@babel/helper-plugin-utils/-/helper-plugin-utils-7.8.3.tgz in my browser.

Can we schedule a video call or exchange contact for faster communication? aaronamankwah2181@gmail.com

**Comment by JeffreyCA** [comment]
Discovered some issues affecting Windows. They are fixed in #6.

To ensure the fixes are applied, please run `pip install -r requirements.txt` after pulling the new changes.
